content/release-notices/v1-1-1-distribution-spec.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "OCI distribution-spec v. 1.1.0 release notice"
+---
+
+**Date: 1/29/2025**
+
+**We are excited to inform you that OCI distribution-spec has released version 1.1.1!**
+
+The [OCI Charter’s IP Policy](https://www.opencontainers.org/about/governance), in Section 8.d., provides for notice of the release of a new version of the OCI specification to be provided to all Members. This notice is a reminder that each release triggers the obligations set forth in the Open Web Foundation Final Specification Agreement (OWFa 1.0) (Patent Only) for all Members on the date 30 days after the date of this notice. Links for your convenience:
+
+- GitHub links for release version 1.1.1: https://github.com/opencontainers/distribution-spec/releases/tag/v1.1.1
+- The OCI Charter and IP Policy: https://www.opencontainers.org/about/governance.
```

Comment:
nit, this link 404s:

```console
$ curl -si https://opencontainers.org/about/governance | head -n1
HTTP/2 404 
```

and going back [at least 5y][1] the opencontainers.org About > Governance menu has linked out to https://github.com/opencontainers/tob/blob/main/CHARTER.md .  Might be worth a grep/replace in the places that talk about https://www.opencontainers.org/about/governance today,  and/or setting up a 301 redirect for anyone using the old URI?

[1]: https://github.com/opencontainers/opencontainers.org/blame/421cdbb95426600cc1422405a2f0968b9b96e559/config.toml#L70

Reply:
ah, looks like I'm doubling up on [the existing report][1], after reading the patch first before getting into pull-request comments 😅  Sorry for the noise!

[1]: https://github.com/opencontainers/opencontainers.org/pull/157#pullrequestreview-2584708980

Reply:
NP, that link is fixed in #158.